app/form_builders/adp_text_field.rb

```diff
@@ -52,7 +52,7 @@ def has_errors?
   def to_html
     result = div_start
     # Issue: duplicate id, this breaks the label & input association
-    result += anchor
+    # result += anchor
```

Comment:
is this removal deliberate? does it not break error message linking?

Reply:
This removal is deliberate as error message were not linked to the relevant input, instead it was linked to an anchor above the input form group. This was an accessibility issue.